README.md

```diff
@@ -4,6 +4,34 @@ A library for interfacing with the Trello API.
 
 Heavily influenced by https://github.com/parroty/extwitter with some stuff straight ripped out of it.
 
+## **Important! Migration from 0.x -> 1.x**
+Since this change will break all existing projects using ExTrello upon upgrading this deserves a spot at the top :)
+
+All calls to the Trello API will now be wrapped in a tuple with the first element either being `:ok`, `:error`, or `:connection_error`
+Regular errors will no longer raise exceptions as that is not idiomatic.
+
+Here's an example(just a little sample):
+```elixir
+# Old and bad
+boards = ExTrello.boards()
+# New and great
+{:ok, boards} = ExTrello.boards()
+
+# Old and bad, yuck!
+try do
+  ExTrello.get("potato/2")
+rescue
+  %ExTrello.Error{code: code, message: message} -> IO.puts "ERROR[#{code}] - #{message}"
+end
+
+# New and fantastic
+case ExTrello.get("potato/2") do
+  {:ok, response} -> response
+  {:error, code, message} -> IO.puts "ERROR[#{code}] - #{message}"
```

Comment:
I think none of them should raise, people should be forced to handle all kinds of errors.

I think removing the code would work. You could possibly add the code inside of the struct for extra debugging info though. Also provides some context and extendability.